diff -up flatdrake-2.2.0/.project.omv flatdrake-2.2.0/.project
--- flatdrake-2.2.0/.project.omv	2024-07-19 07:22:07.000000000 -0400
+++ flatdrake-2.2.0/.project	2025-01-23 16:19:27.545142091 -0500
@@ -4,7 +4,7 @@ Startup=FMAIN
 Icon=flatdrake.svg
 Version=2.2.0
 Component=gb.image
-Component=gb.qt5
+Component=gb.qt6
 Component=gb.form
 Description="FlatDrake è un frontend a FlatPak.\n\nTutto è iniziato partendo dalla base di DnfDrake che parte dal programma di esempio Console per Gambas di Princeton."
 Authors="Autore: Mauro Carbini aka Astragalo\nMail: mauro.carbini@gmail.com"
